builder-bullseye.Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM php-base
+
+COPY --from=golang-base /usr/local/go/bin/go /usr/local/bin/go
+COPY --from=golang-base /usr/local/go /usr/local/go
+
+# This is required to link the frankenPHP binary to the PHP binary
+RUN apt-get update && \
+    apt-get -y --no-install-recommends install \
+        libargon2-dev \
+        libcurl4-openssl-dev \
+        libonig-dev \
+        libreadline-dev \
+        libsodium-dev \
+        libsqlite3-dev \
+        libssl-dev \
+        libxml2-dev \
+        zlib1g-dev \
+        && \
+        apt-get clean
+
+WORKDIR /go/src/app
+
+COPY go.mod go.sum ./
+RUN go mod graph | awk '{if ($1 !~ "@") print $2}' | xargs go get
```

Comment:
Can you add a comment explaining why it's necessary please?

Reply:
Separate file or what?

Reply:
I don't understand what's it's doing :D